http/handler.go

```diff
@@ -136,7 +136,7 @@ func handleRequestForwarding(core *vault.Core, handler http.Handler) http.Handle
 		// Attempt forwarding the request. If we cannot forward -- perhaps it's
 		// been disabled on the active node -- this will return with an
 		// ErrCannotForward and we simply fall back
-		statusCode, retBytes, err := core.ForwardRequest(r)
+		statusCode, header, retBytes, err := core.ForwardRequest(r)
```

Comment:
It's named `header` because of the type: https://golang.org/pkg/net/http/#Header